skills/odl-pdf/scripts/sync-skill-refs.py

```
@@ -0,0 +1,195 @@
+import json
+import re
+import sys
+from pathlib import Path
+
+# Reconfigure stdout to UTF-8 when running on Windows with a legacy code page
+# so that Unicode symbols (checkmark, cross) print correctly in all terminals.
+if hasattr(sys.stdout, "reconfigure"):
+    try:
+        sys.stdout.reconfigure(encoding="utf-8")
+    except Exception:
+        pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)